server/src/main/main.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```

Comment:
What's this?

Reply:
That's an IntelliJ file. Don't know why it got missed by .gitignore

Reply:
Could we do this for .gitignore

```
...
/**/*.iml
...
```

Reply:
Should be able to. I think these showed up because @jasonxris used the other IntelliJ project structure. Shouldn't be an issue to exclude them

Reply:
Yeah, i'll get rid of these. didn't realize they snuck in there